src/plugins/github/porcelain.js

```diff
@@ -162,22 +162,18 @@ class GithubEntity<T: NodePayload> {
 }
 
 export class Repository extends GithubEntity<RepositoryNodePayload> {
-  // TODO: Now that the Repository is a node in the graph, re-write methods
-  // that find issues and PRs to find neighbors of the repository rather than
-  // any matching nodes in the graph. Then, behavior will be correct in the
-  // case where we have multiple repositories in the same graph.
   issueOrPRByNumber(number: number): ?(Issue | PullRequest) {
     let result: Issue | PullRequest;
-    this.graph.nodes({type: ISSUE_NODE_TYPE}).forEach((n) => {
-      if (n.payload.number === number) {
-        result = new Issue(this.graph, n.address);
-      }
-    });
-    this.graph.nodes({type: PULL_REQUEST_NODE_TYPE}).forEach((n) => {
-      if (n.payload.number === number) {
-        result = new PullRequest(this.graph, n.address);
-      }
-    });
+    this.graph
+      .neighborhood(this.nodeAddress, {
+        edgeType: CONTAINS_EDGE_TYPE,
+      })
+      .forEach(({neighbor}) => {
+        const payload = this.graph.node(neighbor).payload;
+        if (payload.number === number) {
```

Comment:
After a bunch of investigation, it looks like this typechecks for Flow versions strictly before v0.68. Fine with me for now.